src/avram/charms/int32_extensions.cr

```diff
@@ -34,6 +34,10 @@ struct Int32
     end
 
     class Criteria(T, V) < Avram::Criteria(T, V)
+      def between(low_value : V, high_value : V)
```

Comment:
Thoughts on BetweenCriteria module that can be included in these?

Reply:
Makes sense